portfolio/index.js

```diff
@@ -0,0 +1,22 @@
+// let music = document.querySelector('source');
```

Comment:
Why do you have extra commented out JS? Purge commented out code before submitting a review request.